### PR TITLE
Added OpenCollective backers and sponsors

### PR DESCRIPTION
**Pull request by xdamman**

### CODE REVIEW

**Comment by jonathanong** [comment]
what's with all the links and images? a lot of the images don't work for me

**Comment by xdamman** [comment]
See https://github.com/OpenCollective/OpenCollective/wiki/Github-banner

**Comment by PlasmaPower** [comment]
@jonathanong They work fine for me.

@xdamman Now that you've got the first backer setup to be a "Become a backer/sponsor" button, I don't think you need the link just before it.

**Comment by PlasmaPower** [comment]
Hmm, badges on GitHub seem to be somewhat broken in general for me - often when I visit the repo there are broken images (network tab says 404), and when I refresh some of them fix themselves while others break. It seems like they have a 50% chance to load or something. I'm not sure why.

File: Readme.md [hunk 1]
```diff
@@ -113,6 +114,78 @@ $ make test
   - [Julian Gruber](https://github.com/juliangruber)
   - [Yiyu He](https://github.com/dead-horse)
 
+
+## Backers
+
+Support us with a monthly donation and help us continue our activities. [[Become a backer](https://opencollective.com/koajs#backer)]
```
Comment (PlasmaPower) [inline Readme.md]: The first backer's badge has been changed to a Become a Backer icon: https://opencollective.com/static/images/become_backer.svg

That makes this link (at the end of the line) repetitive.

Comment (xdamman) [inline Readme.md]: Does that hurt?
It's good for SEO :)

Comment (PlasmaPower) [inline Readme.md]: Then maybe we should remove the first backer icon/link? It just seems a bit pushy.

Comment (xdamman) [inline Readme.md]: Done.

**Comment by xdamman** [comment]
Is this good to merge?

**Comment by PlasmaPower** [comment]
@xdamman Seems good to me. Normally I would say squash the commits, but GitHub has that integrated now.

I don't have push permissions though, so you'll probably have to wait for @jonathanong or @dead-horse.

**Comment by PlasmaPower** [comment]
Should sponsors be before backers? Just wondering.

**Comment by tj** [comment]
LGTM, we can tweak later if necessary

**Pull request merged**